Sources/XcodeGen/main.swift

```diff
@@ -41,7 +41,7 @@ func generate(spec: String, project: String, isQuiet: Bool, justVersion: Bool) {
     do {
         logger.info("⚙️  Generating project...")
         let projectGenerator = ProjectGenerator(project: project)
-        let xcodeProject = try projectGenerator.generateXcodeProject()
+        let xcodeProject = try projectGenerator.generateXcodeProject(xcodeGenVersion: version)
```

Comment:
I agree this can be a useful feature, but at the moment I feel its affects are a bit too pervasive in the codebase. I'd be happy to have an extension `Project.meetsMinimumVersion(_ version: Version) -> Bool`  that gets checked for in this file and throws an error.

XcodeGenKit is also a library that gets used by other packages as well.

Reply:
Sure. I’ll adjust.